acl/authorizer.go

```diff
@@ -21,6 +21,10 @@ const (
 	Default
 )
 
+func (d EnforcementDecision) GoString() string {
```

Comment:
This helps render the actual string value in testify comparison errors instead of their integer values.